app/models/user.rb

```diff
@@ -11,6 +11,14 @@
 #  updated_at    :datetime         not null
 #
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] I think you should remove the included devise modules from lines 16 to 17 since you have the same modules at lines 20 and 21 to make your code clean.

Reply:
done!